### ISSUE TITLE
Cleanup naming issue (setting/settings) and add workspace link

### ISSUE THREAD

**Issue by ankush** [frappe/ecommerce_integrations#106]

**Comment by NagariaHussain** [comment]
Is this this issue @ankush? Users are not able to open it from the dashboard link.

<img width="1440" alt="Screenshot 2021-11-22 at 12 52 12 PM" src="https://user-images.githubusercontent.com/34810212/142820422-bf54f2ad-3748-4dac-b624-29d7394de85e.png">

<img width="664" alt="Screenshot 2021-11-22 at 12 51 08 PM" src="https://user-images.githubusercontent.com/34810212/142820502-f7c8d03d-5961-4edc-9b1c-742cf167f7c5.png">

**Comment by ankush** [comment]
Yep that link is stale link from ERPNext. Need to add a new one.
